tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -144,6 +151,61 @@ def do_create_reports(args: CensusBuildArgs) -> bool:
     return True
 
 
+def do_data_copy(args: CensusBuildArgs) -> bool:
+    """Copy data to S3"""
+    from .data_copy import sync_to_S3
+
+    sync_to_S3(
+        args.working_dir / args.build_tag,
+        urlcat(args.config.cellxgene_census_S3_path, args.build_tag),
+        dryrun=args.config.dryrun,
+    )
+    return True
+
+
+def do_the_release(args: CensusBuildArgs) -> bool:
+    from .release_manifest import CensusVersionDescription, make_a_release
+
+    release: CensusVersionDescription = {
+        "release_date": None,
+        "release_build": args.build_tag,
+        "soma": {
+            "uri": urlcat(args.config.cellxgene_census_S3_path, args.build_tag, "soma/"),
+            "s3_region": "us-west-2",
+        },
+        "h5ads": {
+            "uri": urlcat(args.config.cellxgene_census_S3_path, args.build_tag, "h5ads/"),
+            "s3_region": "us-west-2",
+        },
+    }
```

Comment:
I suppose this `release` declaration could be moved into `make_a_release`, which would allow it to be unit tested.

Reply:
Pushing all of the data layout assumptions down to the module feels like it pollutes the abstraction. I'll add some tests for the __main__.do_* methods instead.

Reply:
added tests for all of the new workflow steps, called from the top-level